modules/http/src/test/scala/higherkindness/mu/rpc/http/Utils.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.rpc.http
+
+import cats.ApplicativeError
+import cats.effect._
+import cats.implicits._
+import fs2.interop.reactivestreams._
+import fs2.{RaiseThrowable, Stream}
+import io.grpc.Status.Code._
+import jawn.ParseException
+import io.circe._
+import io.circe.generic.auto._
+import io.circe.jawn.CirceSupportParser.facade
+import io.circe.syntax._
+import io.grpc.{Status => _, _}
+import jawnfs2._
+import monix.execution._
+import monix.reactive.Observable
+import org.http4s._
+import org.http4s.dsl.Http4sDsl
+import scala.concurrent.ExecutionContext
+import scala.util.control.NoStackTrace
+
+object Utils {
```

Comment:
We need to move this class to the src/main root instead of src/test, since it will be imported in the expanded HTTP macros in production code.

We may also want to turn it into a package object.

Reply:
Done at https://github.com/higherkindness/mu/pull/552/commits/34c05046715f0787bcf7d93554cf29ca0259b594